### PR TITLE
Move requirements to dependency_management

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala/pull/3318/commits/1d36727205f83157e2c2e95d42ff2eaae11acc7d

**Comment by Adrianzatreanu** [comment]
reack e2c4537

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ colorlog~=2.7
 libclang-py3~=3.4.0
 Pygments~=2.1
 PyPrint~=0.2.6
+dependency_management==0.1.5
```
Comment (sils) [inline requirements.txt]: be sure to make a commit to coala to remove the requirement there as it's implicitly in coala from the next prerelease! Can you make an issue

Comment (sils) [inline requirements.txt]: also ==? Not ~=?

**Comment by Adrianzatreanu** [comment]
ack c0c0d0f

**Comment by Adrianzatreanu** [comment]
unack c0c0d0f

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3318/commits/c0c0d0fc82d2a804801753ee348364892264dfc7

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
merging in https://github.com/coala/coala/pull/3330

**Pull request closed without merging**